website/common/locales/en/messages.json

```diff
@@ -56,6 +56,7 @@
   "messageGroupChatFlagAlreadyReported": "You have already reported this message",
   "messageGroupChatNotFound": "Message not found!",
   "messageGroupChatAdminClearFlagCount": "Only an admin can clear the flag count!",
+  "messageGroupChatSpam": "You have been sending messages too quickly, please wait briefly and try again.",
```

Comment:
Thank you! Please let me know if there are any other ideas for how this message could be better